test/functional/page_objects/visualize_page.js

```diff
@@ -656,17 +656,16 @@ export function VisualizePageProvider({ getService, getPageObjects, updateBaseli
     }
 
     async toggleAutoMode() {
-      await testSubjects.click('visualizeEditorAutoButton');
-    }
+      // await testSubjects.click('visualizeEditorAutoButton');
```

Comment:
I guess this commented code should be removed.

Reply:
This code will be uncommented after fixing the issue in eui

Reply:
Got it after reading again :)